src/averageAges.js

```diff
@@ -37,7 +41,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const hasChildren = (name) => {
+    return people.some((human) => human.mother === name);
```

Comment:
I guess looks more logical this way, doesn't it?
```suggestion
    return people.some((person) => person.mother === name);
```